### PR TITLE
Updating Templates and Wizards for 1.2.0 release.

### PR DESCRIPTION
**Pull request by singhsarab**

### CODE REVIEW

File: Templates/CSharp/Desktop/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="mstestProjectTemplate" Version="15.0.26130.00" Language="en-US" Publisher="Microsoft" />
+    <Identity Id="mstestProjectTemplate" Version="15.0.27010.00" Language="en-US" Publisher="Microsoft" />
```
Comment (AbhitejJohn) [inline Templates/CSharp/Desktop/source.extension.vsixmanifest:4]: just curious, where did we get this number from?

**Comment by AbhitejJohn** [comment]
I'm hoping you just ran the update script to get to these changes.

File: Templates/CSharp/Desktop/source.extension.vsixmanifest (hunk 1 above)
Comment (jayaranigarg) [reply Templates/CSharp/Desktop/source.extension.vsixmanifest:4]: Picked up the latest build number of VS

Comment (AbhitejJohn) [reply Templates/CSharp/Desktop/source.extension.vsixmanifest:4]: I see, should it be 15.5 then?

**Pull request merged**